doc/sphinx/addendum/type-classes.rst

```diff
@@ -318,6 +320,10 @@ Summary of the commands
    declared rigid during resolution so that the class abstraction is
    maintained.
 
+   The `>` in
+   :token:`record_definition` currently does nothing. In a future version, it will
+   create coercions as it does when used in :cmd:`Record` commands.
+
```

Comment:
Do we want to mention why it does nothing? I mean it was previously documented to do something but the reader might not know that.

Perhaps something like:
```suggestion
   The `>` in :token:`record_definition` currently does nothing, even
   though it was documented in previous versions. In a future version, it will create
   coercions as it does when used in :cmd:`Record` commands.
```

Reply:
Not sure I get your point here, nor even what "it was documented in previous versions" mean?
Do you mean that we should insist on the fact that the thing was improperly documented before?

Reply:
Right, I just thought it might be confusing for readers to see mention of a no longer existing `>` but I could also be over thinking it. Perhaps it is fine as it is.

Reply:
Well, it reads "currently does nothing", that does not imply anything about the past.
So I think it should be fine (particularly considering this is transitory and will change in some further release).
But thanks for raising the concern anyway.